### ISSUE TITLE
Target netstandard2.0

### ISSUE THREAD

**Issue by Jonarw** [spreadsheetlab/XLParser#176]
It would be nice to target `netstandard2.0` instead of (or additionally to) `netstandard1.6`. This would reduce the number of dependencies package consumers would have to deal with (see https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#net-standard-versions).

I did a quick feasibility check and encountered one problem: The `Irony.NetCore` v1.0.11 dependency targets `netstandard1.6`, so upgrading to 2.0 here would not help.
`Irony.NetCore` v1.1.11 does in fact target `netstandard2.0` (see #175), however it apparently is not signed anymore, so it can't be used here. And anyway the package seems to be inactive (no activity since 2019).

There is an alternative which seems to be more active, targets `netstandard2.0` and is signed: https://github.com/IronyProject/Irony
I did a quick check, this would be a drop-in replacement, all tests pass without further code changes.
The only downside of this would be that XLParser would need to drop `net452` and `netstandard1.6` targets. Not a big loss in my opinion, but not for me to say.

In short, I would propose the following actions:
- Drop reference to [`Irony.NetCore` v1.1.11](https://www.nuget.org/packages/Irony.NetCore)
- Add reference to [`Irony` v1.2](https://www.nuget.org/packages/Irony) instead
- Target `net461` and `netstandard2.0`

Let me know what you think. I'd be happy to make a PR if there is interest.

**Comment by WillemJann** [comment]
Totally agree. If you want to make a PR I'd be happy to review it and to merge this change into XLParser.

**Comment by WillemJann** [comment]
A small addition: because Microsoft [retired .NET Framework 4.6.1](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework), I suggest that we target `net462` and `netstandard2.0`.
